content/documentation/staging/configuration/authentication/openid.adoc

```diff
@@ -274,3 +274,11 @@ spec:
       http_proxy: http://USERNAME:PASSWORD@10.0.1.1:8080/
       https_proxy: https://USERNAME:PASSWORD@10.0.0.1:8080/
 ----
+
+=== Usage with Keycloak
```

Comment:
Maybe change to "Using Keycloak" for consistency with the section headers above.